src/main/java/lifeline/task/TaskList.java

```diff
@@ -0,0 +1,108 @@
+package lifeline.task;
+
+import java.util.ArrayList;
+
+import lifeline.exception.LifelineException;
+
+/**
+ * The class TaskList encapsulates a list of tasks using an ArrayList of tasks
+ */
+public class TaskList {
+
+    private ArrayList<Task> taskList;
+
+    /**
+     * Default constructor for a TaskList
+     *
+     * @param taskList List of tasks to be stored
+     */
+    public TaskList(ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Adds a task to TaskList
+     *
+     * @param task Task to be added
+     */
+    public void add(Task task) {
+        taskList.add(task);
+    }
+
+    /**
+     * Returns task based on given index
+     *
+     * @param index Index of task to be returned
+     * @return Task at specified index
+     */
+    public Task get(int index) {
```

Comment:
Maybe could use the name getTask rather than get to better following the naming convention.

Reply:
I was trying to follow the arrayList convention but you're right it should be named getTask. Thanks!